src/main/java/com/j256/ormlite/dao/CloseableSpliteratorImpl.java

```diff
@@ -0,0 +1,49 @@
+package com.j256.ormlite.dao;
+
+import com.j256.ormlite.misc.IOUtils;
+
+import java.util.Spliterator;
+import java.util.Spliterators;
+import java.util.function.Consumer;
+
+public class CloseableSpliteratorImpl<T> implements CloseableSpliterator<T> {
```

Comment:
Can we add a comment here?  Yeah it's pretty obvious what this is doing but it's a pattern.